ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/host1.md

```diff
@@ -815,10 +815,12 @@ management accounts
 
 #### Management API gNMI Summary
 
-| Transport | SSL Profile | VRF | Notification Timestamp | ACL | Port |
-| --------- | ----------- | --- | ---------------------- | --- | ---- |
-| MGMT | gnmi | MGMT | send-time | acl1 | 6030 |
-| mytransport | - | - | send-time | acl1 | 6032 |
+| Transport | SSL Profile | VRF | Notification Timestamp | ACL | Port | Authorization Requests |
+| --------- | ----------- | --- | ---------------------- | --- | ---- | ---------------------- |
+| MGMT | gnmi | MGMT | send-time | acl1 | 6030 | false |
+| mytransport | - | - | send-time | acl1 | 6032 | false |
```

Comment:
You are defaulting to a string "false", but you also have booleans, which makes this look weird. false, False, True etc...